lib/event_factory.rb

```diff
@@ -74,32 +74,6 @@ def self.project_refund_request(project, user, reference)
     # EventfulMailer.deliver_confirm_event(User.all_administrators_emails, event.eventful, event.message, event.content, "No Milestone")
   end
 
-  ###############################
```

Comment:
Found this when looking for registration associated stuff. Doesn't look like it was actually triggered there, and looks like whatever was calling it was removed a long time ago.